Model/Venmo/Ui/ConfigProvider.php

```diff
@@ -0,0 +1,147 @@
+<?php
+declare(strict_types=1);
+
+namespace Magento\Braintree\Model\Venmo\Ui;
+
+use Magento\Braintree\Gateway\Request\PaymentDataBuilder;
+use Magento\Braintree\Model\Adapter\BraintreeAdapter;
+use Magento\Braintree\Model\Venmo\Config;
+use Magento\Checkout\Model\ConfigProviderInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Exception\InputException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\View\Asset\Repository;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Class ConfigProvider
+ *
+ * @package Magento\Braintree\Model\Venmo\Ui
```

Comment:
package and the author shouldn't be used

Reply:
Bad habit 😂